### ISSUE TITLE
Devise confirmation emails

### ISSUE THREAD

**Issue by bdougie** [rapid99/Blocipedia#2]
> I am having issues setting up the confirmation emails (would like to go over this next meeting). You can sign in successfully and access profile views. I need to create an error or alert of some sort to pop up if invalid credentials are entered as it currently just reloads the page. If you try to sign up with a duplicate email, you get an error stating the email has already been taken

Projects now are more complex and impossible to debug with minimal information, please open issues from now on and provide more details on the situation

**Comment by bdougie** [comment]
https://github.com/rapid99/Blocipedia/blob/checkpoint-2/app/models/user.rb#L3

Take a look at this link https://github.com/plataformatec/devise/wiki/How-To:-Add-:confirmable-to-Users

**Comment by bdougie** [comment]
Re-reading and looks like you need help creating error messages: https://coderwall.com/p/suyeew/devise-error-messages-twitter-bootstrap-style

**Comment by rapid99** [comment]
Hey Brian, 

I went ahead and reinstalled Devise after rolling back to a previous commit (before accidentally making that model named "model". I went through the instructions for installing Letter Opener as well but still can't get it to work properly. I commited the branch "devise" which has my current state on it. 

I'm going to move forward with the CRUD actions now to get started on this. Let me know what you think
